debug_exception.py

```
@@ -0,0 +1,64 @@
+
+import asyncio
+from mad_spark_alt.core import SmartQADIOrchestrator
+
+async def test():
+    orchestrator = SmartQADIOrchestrator()
+    
+    try:
+        await orchestrator.ensure_agents_ready()
+        print("✅ Agents ready")
+        
+        result = await orchestrator.run_qadi_cycle(
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)